### PR TITLE
fix: null value in deployment deployment log

### LINKED ISSUE #317: `null` in deployment_log table
In the `deploy_log` table, `targets` and `sources` for deployment of type _DAG_ are set to str `"null"` instead of null value.

### PR DESCRIPTION
**Pull request by jpsavy**
#### Which issue(s) this PR fixes

Fixes #317 

#### Additional comments

The null value in the target and source columns of the deployment_log table is changed to JSON.NULL < null > after running the command: tdp plan dag

#### Agreements

- [X] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [X] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Review by PaulFarault — APPROVED** [review]
It has been triggering me for a while. Nice catch!

**Review by rpignolet — CHANGES_REQUESTED** [review]
For me this is not the right way to correct. You have to change the schema so that in the case of a `None` the "SQL null" is used.
Here is what I found on the internet: https://stackoverflow.com/questions/29309696/filter-postgres-json-column-by-null-value-in-sqlalchemy
If you can look to add the `none_as_null=True` at schema level to test.

**Comment by jpsavy** [comment]
> For me this is not the right way to correct. You have to change the schema so that in the case of a `None` the "SQL null" is used. Here is what I found on the internet: https://stackoverflow.com/questions/29309696/filter-postgres-json-column-by-null-value-in-sqlalchemy If you can look to add the `none_as_null=True` at schema level to test.

According to SQlAlchelmy documentation : 
https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.JSON.params.none_as_null
Using Column(JSON(none_as_null=True)) or Column(JSON(none_as_null=True)) for the sources ad targets columns in the DeploymentLog class doesn't work : always null instead of <null> for a 'None' value.

This is why a use this actual workaround : 
targets=null() if targets is None else targets,

File: tdp/core/deployment/deployment_plan.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Copyright 2022 TOSIT.IO
 # SPDX-License-Identifier: Apache-2.0
 
+from sqlalchemy import null,text
```
Comment (rpignolet) [inline tdp/core/deployment/deployment_plan.py:4]: Why adding this import ?

Comment (jpsavy) [reply tdp/core/deployment/deployment_plan.py]: > Why adding this import ?

No more import null and text from sqlalchelemy in deployment_plan.py

**Comment by rpignolet** [comment]
LGTM, if you can squash your commit please. Not need to have 3 commits to change 2 lines.

**Comment by PaulFarault** [comment]
I'll do it for @jpsavy as he won't be available for a few weeks.

**Review by PaulFarault — APPROVED** [review]

**Review by rpignolet — APPROVED** [review]

**Pull request merged**